docs/public-networks/how-to/connect/sync-node.md

```diff
@@ -116,8 +116,13 @@ Checkpoint sync behaves like [snap sync](#snap-synchronization), but instead of
 genesis block, it syncs from a specific checkpoint block configured in the [Besu genesis
 file](../../concepts/genesis-file.md).
 
-You can configure a checkpoint in the genesis file by specifying the block hash, number, and total
-difficulty as in the following example.
+Checkpoints are currently already defined in the network configurations for Ethereum Mainnet and
```

Comment:
Could we simplify this sentence with something like "Ethereum Mainnet and Goerli testnet already define default checkpoints. If you use these predefined networks, only add [the `--sync-mode=X_CHECKPOINT` option](../../reference/cli/options.md#sync-mode) for checkpoint synchronisation."

Reply:
simplified!